starknet_py/net/rpc_schemas/rpc_schemas.py

```diff
@@ -140,8 +139,10 @@ def make_dataclass(self, data, **kwargs) -> DeclareTransaction:
 
 class DeployTransactionSchema(TransactionSchema):
     contract_address = Felt(data_key="contract_address", required=True)
-    constructor_calldata = fields.List(Felt(), data_key="calldata", required=True)
-    # class_hash = Felt(data_key="class_hash", load_default=None)
+    constructor_calldata = fields.List(
+        Felt(), data_key="constructor_calldata", required=True
+    )
+    class_hash = Felt(data_key="class_hash", load_default=None)
```

Comment:
I wonder if accepting `None` as class_hash is a good idea. This seems like the most important field in the `DeployTransaction`.

Reply:
Perhaps we should get rid of `load_default` here entirely.